libp2p/Host.cpp

```diff
@@ -667,22 +672,7 @@ size_t Host::peerCount() const
 void Host::run(boost::system::error_code const&)
```

Comment:
I think we should better check this error code, so that it doesn't proceed when io_service is stopped.